contracts/interfaces/IServiceRegistry.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
-pragma solidity 0.8.13;
+pragma solidity 0.8.19;
 
 interface IServiceRegistry {
     function getRegisteredService(string memory) external view returns (address);
+    function getServiceAddress(bytes32) external view returns (address);
```

Comment:
I know this is mimicking  the `dma-contracts` interface, but if we have the opportunity to rename it, I think it is better to call it `getRegisteredServiceByHash`